### PR TITLE
Added translation es_ES, for Spanish (Spain).

### PR DESCRIPTION
**Pull request by lcabrera**
This translation is for Spanish in Spain.

### CODE REVIEW

File: Europe/es_ES.json [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+  "Notes": "Notas",
+  "Reminder": "Recordatorio",
+  "Supplies": "Suministros",
+  "Planner": "Planificador",
+  "Default_Tab": "Pestaña predeterminada",
+  "Service_Record": "Hoja de servicios",
+  "Tax": "Impuesto",
+  "Reminders": "Recordatorios",
+  "Backups": "Copias de seguridad",
+  "Make": "Haz",
```
Comment (nanitook) [inline Europe/es_ES.json:37]: this has already being merged in the main repo, but this translation is used in two different places with different meanings in languages like spanish (car manufacturer and create backup button)
consider changing the translation

File: Europe/es_ES.json [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+  "No_supplies_with_quantities_greater_than_0_is_found.": "No se han encontrado suministros con cantidades superiores a 0.",
+  "Select": "Seleccionar",
+  "#_of_Upgrade_Records": "N.º de registros de actualización",
+  "Add_Upgrade_Record": "Añadir registro de actualización",
+  "Add_New_Upgrade_Record": "Añadir nuevo registro de actualización",
+  "Date_upgrade/mods_was_installed": "Fecha de instalación de la actualización/modificación",
+  "Odometer_reading_when_upgraded/modded": "Lectura del cuentakilómetros cuando se actualiza/modifica",
+  "Description_of_item(s)_upgraded/modded": "Descripción de los elementos mejorados/modificados",
+  "Cost_of_the_upgrade/mods": "Coste de la actualización/modificación",
+  "#_of_Gas_Records": "N.º de registros de ga",
```
Comment (nanitook) [inline Europe/es_ES.json:131]: abbreviation?

File: Europe/es_ES.json [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+  "Reminder_Description": "Descripción del recordatorio",
+  "Remind_me_on": "Recuérdamelo en",
+  "Future_Date": "Fecha futura",
+  "Future_Odometer_Reading": "Futura lectura del cuentakilómetros",
+  "Whichever_comes_first": "Lo que ocurra antes",
+  "Other": "Otros",
+  "Edit_Reminder": "Editar el recordatorio",
+  "Replace_picture(optional)": "Sustituir la imagen (opcional)",
+  "Language": "Idioma",
+  "Manage_Languages": "Manage Languages",
```
Comment (nanitook) [inline Europe/es_ES.json:220]: not translated

File: Europe/es_ES.json [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+  "Back_to_Login": "Volver al inicio de sesión",
+  "Email_Address": "Dirección de correo electrónico",
+  "New_Password": "Nueva contraseña",
+  "Reset_Password": "Restablecer contraseña",
+  "No_data_found_or_all_records_have_zero_sums,_insert_records_with_non-zero_sums_to_see_visualizations_here.": "No se han encontrado datos o todos los registros tienen sumas cero, inserte registros con sumas distintas de cero para ver las visualizaciones aquí.",
+  "Save_as_Template": "Guardar como plantilla",
+  "View_Templates": "Ver plantillas",
+  "Select_Template": "Seleccionar plantilla",
+  "No_templates_are_found.": "No se han encontrado plantillas.",
+  "Use": "Use",
```
Comment (nanitook) [inline Europe/es_ES.json:248]: I think this may be referring to a verb like "Usar" or "Utilizar"

**Review by nanitook — CHANGES_REQUESTED** [review]
made some comments on things I think you may need to take a look at

File: Europe/es_ES.json (hunk 1 above)
Comment (lcabrera) [reply Europe/es_ES.json:37]: I am sorry to say that I do not understand this point.

Can you reference the line number that you think is conflicting?

File: Europe/es_ES.json (hunk 2 above)
Comment (lcabrera) [reply Europe/es_ES.json:131]: First thank you for the review :+1: .

It is not an abbreviation, it is simply a mistake. Sorry.

...but it's good for me to hear a second opinion.

In the original translation, the word `gas` is used, but I think I understand that it actually refers to the word `fuel` (`combustible` in Spanish).

Did I misunderstand?

File: Europe/es_ES.json (hunk 3 above)
Comment (lcabrera) [reply Europe/es_ES.json:220]: Fixed

File: Europe/es_ES.json (hunk 4 above)
Comment (lcabrera) [reply Europe/es_ES.json:248]: ok, let's use (`usar`).

**Comment by lcabrera** [comment]
> made some comments on things I think you may need to take a look at

I have prepared a pull request with the exception of the first comment.

As soon as we see how to correct the first comment, I will prepare another pull request for it.

**Comment by lcabrera** [comment]
I have prepared a pull request with the exception of the first comment.

I have added some minor corrections, with the intention of making the translation more consistent.

File: Europe/es_ES.json (hunk 2 above)
Comment (nanitook) [reply Europe/es_ES.json]: No problem :)

Yes, translating to "combustible" it's ok

File: Europe/es_ES.json (hunk 1 above)
Comment (nanitook) [reply Europe/es_ES.json:37]: Line 37. 
I think it's already changed in the en_us original file, but this translation used to be used in different locations with different meanings, one being the `car manufacturer` ("marca o fabricante"), and the other one being the `create backup button` ("hacer").
As stated [here](https://github.com/hargata/lubelog_translations/pull/10#issuecomment-1937079301) "Make" is now used just to refer to the car manufacturer and "Create" (new translation) is used to refer to the `create backup button`.

So, this one should be changed to something like "fabricante".

Comment (lcabrera) [reply Europe/es_ES.json]: ok, fixed.

**Comment by lcabrera** [comment]
Fixed from "Haz" to "Fabricante".

**Review by nanitook — APPROVED** [review]

**Pull request merged**